docsite/docs/Reference/fal-scripts-path.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 1
+---
+
+# Variable `fal-scripts-path`
+
+fal uses the `fal-scripts-path` [dbt variable](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-in-dbt_projectyml) to scripts in your project.
+
+Let's consider the following file structure for some examples:
+```
+.
+├── dbt_project.yml
+├── models
+│   ├── schema.yml
+│   └── some_model.sql
+└── scripts
+    ├── after.py
+    ├── before.py
+    └── utils
+        ├── my_utils.py
+        └── process
+            └── process_df.py
+```
+
+By default `fal-scripts-path` is the [dbt project directory](https://docs.getdbt.com/reference/dbt_project.yml) (where the `dbt_project.yml` is located). 
+
+But it can be chagend in the [vars](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-in-dbt_projectyml). An example of setting the variable would be:
+```yaml
+name: "fal_test"
+version: "1.0.0"
+config-version: 2
+
+vars:
+  fal-scripts-path: "scripts"
+
+...
+```
+
+## Script reference in `schema.yml`
+
+The Python scripts to be [associated to your models](../Docs/fal-cli/model-scripts.md) use the `fal-scripts-path` dbt variable as a base directory to search the files.
+
+Meaning that referencing a script in your `schema.yml` with the default `fal-scripts-path` value would look like:
```

Comment:
Referencing a script in your `schema.yml` with the default `fal-scripts-path` value looks like: